workflow/signals.py

```diff
@@ -204,15 +204,7 @@ def check_seats_save_team(sender, instance, **kwargs):
     except APIError as e:
         logger.warn(e)
     else:
-        # Validate the amount of available seats based on the subscription
-        available_seats = subscription.plan_quantity
-        if org.chargebee_used_seats > available_seats and user_gained_seat:
-            extra_context = {
-                'used_seats': used_seats,
-                'available_seats': available_seats,
-                'payment_portal_url': settings.PAYMENT_PORTAL_URL
-            }
-            notify_excess_org_admin(org, extra_context)
+        pass
```

Comment:
You can remove the else as well. There is no need for `else`

Reply:
Done.